fastlane/lib/fastlane/actions/create_keychain.rb

```diff
@@ -126,8 +126,8 @@ def self.available_options
                                        type: Boolean,
                                        default_value: false),
           FastlaneCore::ConfigItem.new(key: :timeout,
-                                       description: 'timeout interval in seconds',
-                                       type: Integer,
+                                       description: 'timeout interval in seconds. Set `false` if you want to specify "no time-out"',
+                                       skip_type_validation: true, # allow integers & Bool->`false` for "no timeout"
```

Comment:
Would it allow String values that are converted into int values or it will crash unexpectedly?

Reply:
Thank you for your review! It will allow and it will not crash, please see this https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/create_keychain.rb#L50

- `timeout: "5000"` & `timeout: 5000` both are equalivent and valid without any crash! 
- `timeout: false` then it will ignore `-t` param

Reply:
I _believe_ no timeout will work if you set `0` as the value 🤔 `0` is also seen as falsey so the `-t` parameter will get skipped. I think we keep this as a `type: Integer` and update the description to say something like `Set `0` if you want to specify “no timeout”`.

I’m pretty sure `0` is a common value to set for no timeout in other libraries/dependencies 🤷‍♂️ 

Thoughts? 😊

Reply:
Thank you very much for this suggestion, on it right away!